lib/reactotron-react-js/package.json

```
@@ -0,0 +1,126 @@
+{
+  "name": "reactotron-react-js",
+  "version": "2.1.1",
+  "description": "A development tool to explore, inspect, and diagnose your React JS/DOM apps.",
+  "author": "Infinite Red",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/infinitered/reactotron/issues"
+  },
+  "homepage": "https://github.com/infinitered/reactotron-react-js",
+  "repository": "https://github.com/infinitered/reactotron-react-js",
+  "files": [
+    "dist",
+    "LICENSE",
+    "README.md",
+    "reactotron-react-js.d.ts"
+  ],
+  "main": "dist/index.js",
+  "types": "./dist/types/index.d.ts",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch --notify",
+    "format": "prettier --write {**,.}/*.ts",
```

We need `yarn format:check` or else `nx` won't pick it up

```suggestion
    "format": "prettier '*.{js,ts,tsx,json,md,css,yml}|**/*.{js,ts,tsx,json,md,css,yml}'",
    "format:check": "yarn format --check",
    "format:write": "yarn format --write",
```